alert-common/src/main/java/com/synopsys/integration/alert/common/rest/ProxyManager.java

```diff
@@ -113,4 +115,16 @@ public ProxyInfo createProxyInfo() throws IllegalArgumentException {
         return settingsConfiguration.flatMap(configurationModel -> configurationModel.getField(key)).flatMap(ConfigurationFieldModel::getFieldValue);
     }
 
+    public Proxy createProxy() {
+        if (!getProxyHost().isPresent()) {
```

Comment:
Why not `isEmpty()`? SonarCloud?

Reply:
Yeah not sure if the SonarCloud issues has been updated yet.